.github/workflows/modular-arch-quality-gates.yml

```
@@ -0,0 +1,644 @@
+              echo "   - $check"
+            done
+          fi
+
+      - name: Run Available Tests with Coverage
+        id: run-tests
+        run: |
+          MODULE_PATH="frontend/packages/${{ matrix.module }}"
+          FRONTEND_PATH="$MODULE_PATH/frontend"
+          BFF_PATH="$MODULE_PATH/bff"
+          HAS_COVERAGE=false
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)